### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by cdiazgil**
Ok many changes in terms of implementing the new individual model and making it work properly
s

### CODE REVIEW

**Comment by ritxart** [comment]
Has upload the data, do you think you wanna upload it?

File: www/D.annularis.csv [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+6973;2.38089;3
+5803;1.46252;1
+5803;2.13359;2
+5803;2.50498;3
+5800;1.66105;1
+5800;2.26135;2
+5800;2.47582;3
+5800;2.63909;4
+5799;1.55571;1
+5799;2.25847;2
```
Comment (ritxart) [inline www/D.annularis.csv]: What is this file?

File: www/S.porcus.csv [hunk 2]
```diff
@@ -0,0 +1,1043 @@
...
+3.167200575;4;17;531
+3.745055752;5;17;531
+1.647976579;1;18;532
+2.297961795;2;18;532
+2.88967551;3;18;532
+3.566801572;4;18;532
+2.05078984;1;19;2181
+2.602478067;2;19;2181
+3.364201427;3;19;2181
+3.69756155;4;19;2181
```
Comment (ritxart) [inline www/S.porcus.csv:57]: ido

File: www/D.annularis.csv (hunk 1 above)
Comment (cdiazgil) [inline www/D.annularis.csv]: it's just the example file

**Comment by cdiazgil** [comment]
Now what is left is to clean the code (there are parts that are just informative)

**Pull request closed without merging**